fhir-bulkimportexport-webapp/WebContent/WEB-INF/web.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<web-app xmlns="http://xmlns.jcp.org/xml/ns/javaee" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_3_1.xsd" version="3.1">
+	<display-name>IBM FHIR BulkImportExport WebAPP</display-name>
+	<welcome-file-list>
```

Comment:
I think we can remove the `welcome-file-list`

Reply:
good point, done!